src/NodaTime/Text/DurationPattern.cs

```diff
@@ -25,17 +25,26 @@ public sealed class DurationPattern : IPattern<Duration>
     {
         /// <summary>
         /// Gets the general pattern for durations using the invariant culture, with a format string of "-D:hh:mm:ss.FFFFFFFFF".
-        /// This pattern round-trips.
+        /// This pattern round-trips. This corresponds to the "o" standard pattern.
         /// </summary>
         /// <value>The general pattern for durations using the invariant culture.</value>
         public static DurationPattern Roundtrip => Patterns.RoundtripPatternImpl;
 
+        /// <summary>
+        /// Gets a pattern for durations using the invariant culture, with a format string of "-H:mm:ss.FFFFFFFFF".
+        /// This pattern round-trips, and is the pattern that NodaTime.Serialization.JsonNet uses by default.
+        /// This corresponds to the "j" standard pattern.
+        /// </summary>
+        /// <value>The pattern for durations using the invariant culture. This is the pattern that NodaTime.Serialization.JsonNet used by default.</value>
```

Comment:
"uses by default"?

Reply:
Humbug - I fixed the first one of those and force-pushed, but missed the second :) Will fix and merge. Thanks!